v22.2/backup-and-restore-overview.md

```diff
@@ -20,20 +20,19 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
 
 {% include cockroachcloud/ccloud/backup-types.md %} 
 
-## Backup and restore types
+## Backup and restore product support
 
-The following table outlines the available backup and restore types in CockroachDB. See each of the pages linked in the table for usage examples:
+This table outlines the level of product support for backup and restore features in CockroachDB. See each of the pages linked in the table for usage examples:
```

Comment:
If you adjust "level of product support" per Liv's earlier comment, also adjust here. I could go either way: "level of product support" being pretty common in the industry, but Liv's direct explanation conveys clearly. Up to you.